### DISCUSSION TITLE
写一篇关于中国 LGBTQIA 群体的文章?

### DISCUSSION THREAD

**Discussion by Michael18811380328** [LGBT-CN/LGBTQIA-In-China#100] [Q&A] [answered]
这个需要在第三方网站上写，还是直接以 markdown 的格式提交？内容有无限制？

**Comment by KevinZonda** [comment, marked as answer]
都可以！但是推荐提供一份Markdown备份！

**Reply by KevinZonda** [reply]
如果只想提供 Markdown，可以参考我的
如果想提供第三方网站，可以参考 Eltrac Koalar 的
但是都需要明确版权信息！

具体描述参见：

https://github.com/LGBT-CN/LGBTQIA-In-China/tree/master/post

版权标注可参考：
https://github.com/LGBT-CN/LGBTQIA-In-China/blame/master/post/%E7%9F%A5%E5%90%A6%EF%BC%9F%E7%9F%A5%E5%90%A6%EF%BC%9F%E5%BA%94%E6%98%AF%E7%BB%BF%E8%82%A5%E7%BA%A2%E7%98%A6.md#L41-L42
